scripts/vaccination_alerts/send-emails.ts

```diff
@@ -0,0 +1,206 @@
+import _ from 'lodash';
+import admin from 'firebase-admin';
+import { getFirestore } from '../common/firebase';
+import EmailService, {
+  EmailSendData,
+  isInvalidEmailError,
+} from '../alert_emails/email-service';
+import * as yargs from 'yargs';
+import { DEFAULT_ALERTS_FILE_PATH, readVaccinationAlerts } from './utils';
+import { RegionVaccinePhaseInfo } from '../../src/cms-content/vaccines/phases';
+
+const BATCH_SIZE = 20;
+
+class SendVaccinationAlertsService {
+  db: FirebaseFirestore.Firestore;
+  emailService: EmailService;
+
+  invalidEmailCount: number = 0;
+  errorCount: number = 0;
+  emailSent: number = 0;
+
+  uniqueEmailAddress: { [email: string]: number } = {};
+  locationsWithEmails: { [fips: string]: number } = {};
+
+  constructor(db: FirebaseFirestore.Firestore, emailService: EmailService) {
+    this.db = db;
+    this.emailService = emailService;
+  }
+
+  private async onEmailSent(email: string, alert: RegionVaccinePhaseInfo) {
+    this.updateSentEmailCounters(email, alert);
+    const docPath = `${alert.fips}/emailVersions/${alert.emailAlertVersion}/emails/${email}`;
+    return this.db
+      .collection('vaccination-alerts')
```

Comment:
Ideally we'd have constants for all of these collection names, but can punt for now.

Also it's a bit arbitrary that we do `collection('vaccination-alerts').doc(docPath)` instead of just including `vaccination-alerts/` at the beginning of `docPath` or doing `.collection('vaccination-alerts').doc(alert.fips).collection('emailVersions')...`